client/fingerprint/env_aws_test.go

```diff
@@ -268,48 +142,99 @@ func TestNetworkFingerprint_AWS_network(t *testing.T) {
 		request := &FingerprintRequest{Config: cfg, Node: node}
 		var response FingerprintResponse
 		err := f.Fingerprint(request, &response)
-		if err != nil {
-			t.Fatalf("err: %v", err)
-		}
+		require.NoError(t, err)
 
 		assertNodeAttributeContains(t, response.Attributes, "unique.network.ip-address")
 
-		if response.NodeResources == nil || len(response.NodeResources.Networks) == 0 {
-			t.Fatal("Expected to find Network Resources")
-		}
+		require.NotNil(t, response.NodeResources)
+		require.Len(t, response.NodeResources.Networks, 1)
 
 		// Test at least the first Network Resource
 		net := response.NodeResources.Networks[0]
-		if net.IP == "" {
-			t.Fatal("Expected Network Resource to have an IP")
-		}
-		if net.CIDR == "" {
-			t.Fatal("Expected Network Resource to have a CIDR")
-		}
-		if net.Device == "" {
-			t.Fatal("Expected Network Resource to have a Device Name")
-		}
-		if net.MBits != 10 {
-			t.Fatalf("Expected Network Resource to have speed %d; got %d", 10, net.MBits)
-		}
+		require.NotEmpty(t, net.IP, "Expected Network Resource to have an IP")
+		require.NotEmpty(t, net.CIDR, "Expected Network Resource to have a CIDR")
+		require.NotEmpty(t, net.Device, "Expected Network Resource to have a Device Name")
+		require.Equal(t, 10, net.MBits)
 	}
 }
 
-func TestNetworkFingerprint_notAWS(t *testing.T) {
```

Comment:
I removed this test as it's exactly the same as `TestEnvAWSFingerprint_nonAws` test above.